pallets/msa/src/lib.rs

```diff
@@ -275,6 +319,36 @@ pub mod pallet {
 		ProofHasExpired,
 		/// The submitted proof expiration block is too far in the future
 		ProofNotYetValid,
+		/// Attempted to add a signature when the signature is already in the registry
+		BucketKeyExists,
```

Comment:
I wonder if a better name is just `PreviouslyUsedSignature`. Or `SignatureReplayDetected` so it is clear to the end user what went wrong.

Reply:
Yea the name is left over from the prototype, I was thinking "SignatureAlreadySubmitted"